### ISSUE TITLE
[Accessibility] A11y_.Net Editor_Screen Reader: Narrator/NVDA is not reading the short keys inside the command palette

### ISSUE THREAD

**Issue by Tanushagottipati** [dotnet/try#108]
**Environment Details:**
https://docs.microsoft.com/en-us/dotnet/csharp/quick-starts/hello-world?tutorial-step=4User 
**Browser:** Edge (41.16299.402.0)
  Chrome (67.0.3396.87)
**Screen Reader:** Narrator
		NVDA

**Additional Details:**
MAS Violated: MAS1.3.1

**Repro steps:**
1. Hit the URL https://docs.microsoft.com/en-us/dotnet/csharp/quick-starts/hello-world?tutorial-step=4User 
2. Tab till “Enter focus mode” and press enter
3. Press F1 and open the command palette
4. Verify Whether the screen reader is reading the short keys and check whether it is accessible by the keyboard.

**Actual Result:**
Screen reader is not reading the short keys inside the command palette 

**Expected Result:**
Screen reader should read the short keys for the user who wants to access it 

**User Impact:**
Screen readers users find difficult in accessing the keyboard short keys if screen reader does not mention them

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}  

[screen reader is not reading the short keys inside the command palette.pptx](https://github.com/dotnet/try/files/2188604/screen.reader.is.not.reading.the.short.keys.inside.the.command.palette.pptx)

**Comment by jonsequitur** [comment]
@Tanushagottipati Please reopen this issue under https://github.com/Microsoft/monaco-editor.

**Comment by LadyNaggaga** [comment]
1.	Importance of experience (High or Low Impact). 
Impact: Low we haven't received any customer complaints directly to the service. The .NET online editor is the service behind editor and we do not own or control the UI experience. 
Describe scenario: Please the description above
2.	Quality of Experience (High or Low Impact). 
Impact: Low we haven't received any customer complaints directly to the service. The .NET online editor is the service behind editor and we do not own or control the UI experience. 
3.	[Link to AAT ](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/93711e75-9cb7-4061-b1e9-53177a087cae)
4.	Any alternate solutions or technologies evaluated
Not applicable 
5.	Ensure all evidence for hand-off to external team is provided in the bug
Issue has been opened on the Monaco repo.

**Comment by umasarath52** [comment]
This was discussed with Ryan and approved to close as this is non blocking scenario.
